app/api/routes.py

```diff
@@ -179,6 +184,57 @@ def get_resources():
     return standardize_response(payload=dict(data=resource_list, **pagination_details))
 
 
+def search_results(term):
+    query = Resource.query
+    page = 0
+
+    if request.args.get('page') is not None:
+        page = int(request.args.get('page'))
+
+    indicies = client.list_indices()
+    for ind in indicies['items']:
+        if ind['name'] == 'resources_api':
+            curr_index = ind
+
+            if curr_index['entries'] != query.count():
+                print("ReIndexing Database")
+
+                index.clear_objects()
+                db_list = [u.serialize for u in query.all()]
+                index.save_objects(db_list)
+
+            break
+
+    search_result = index.search(f'{term}', {
+        'attributesToRetrieve': [
+            'name',
+            'url',
+            'category',
+            'languages',
+            'notes',
+        ],
```

Comment:
If we're just going to get the data from the database here https://github.com/OperationCode/resources_api/blob/b6104234e3e5e2004b7cb285747815a44aaf29fc/app/api/routes.py#L225
then all we need from algolia is the objectid, right? Are we actually doing anything with these attributes?